docs/how-to-guides/s3.rst

```diff
@@ -0,0 +1,54 @@
+.. _s3:
+
+Mounting S3 Buckets in Renku Sessions
+=====================================
+
+Renku has an optional feature that allows users to mount and access data in
+any S3-compatible object storage. This feature is enabled at `renkulab.io <http://renkulab.io>`_ 
+but please note that on other Renku deployment this may not be the case.
+If your deployment does not have this feature enabled you should contact
+your administrator.
+
+Amazon AWS initially came up with the S3 API for object storage. However, nowadays
+almost all cloud providers have S3-compatible object storage. For example, Google Cloud Storage
+is S3 compatible and can be used in Renku. Please refer to your cloud provider documentation
+for more details.
+
+Step-by-step Instructions 
+-------------------------
+
+There are many public datasets that are hosted by AWS or other cloud providers on
+S3-compatible object storage. One such dataset is the `Genome in a Bottle (GIAB) <https://registry.opendata.aws/giab/>`_ 
+dataset hosted on AWS. If you wish to use the GIAB data in a Renku session follow these steps:
+
+1. Navigate to the ``Sessions`` tab in your Renku project.
+2. Click the green ``New Session`` button on the right side of the page.
+3. Click on the ``Do you want to select the branch, commit, image or configure cloud storage`` link.
+4. Click on the ``Configure Cloud Storage`` button in the section that was revealed in the previous step.
+5. In the form that opened, enter the ``Endpoint`` for the GIAB bucket as ``http://s3.amazonaws.com``.
+6. The only other field you have to fill in is the ``Bucket Name`` field. For the GIAB bucket this is simply ``giab``. The ``Access Key`` and ``Secret Key`` fields can be left empty because this is a public bucket.
+7. Click ``Save``.
+8. Click ``Start session``.
+9. Once you session is ready the bucket will be available inside your session at ``/cloudstorage/<bucket name>``. In this case for the GIAB bucket it will be available at ``/cloudstorage/giab``.
+
+Limitations
+-----------
+
+There are a few limitations of this feature that users should be aware of. Please note that as we 
+further develop this feature some of these limitations will be relaxed or fully lifted.
+
+* **Accessing data in mounted buckets is 3x to 10x slower than accessing data on disk.** 
```

Comment:
I would just say "potentially much slower" since the details will depend on where the S3 is located. It could be at switch, in which case it's probably the same performance as "local" disk in the sessions.